src/util/logger.ts

```diff
@@ -23,7 +23,7 @@ export enum LogLevel {
   Fatal,
 }
 
-let level = LogLevel.Info;
+let level = LogLevel.Error;
```

Comment:
It would be good to adjust the default log level to `LogLevel.Warn`. It might captures warnings and potential issues without overwhelming.

```suggestion
let level = LogLevel.Warn;
```

Reply:
Oh I see, I'll make the change and push it again.